cloudinit/sources/__init__.py

```diff
@@ -302,6 +302,11 @@ def persist_instance_data(self):
         instance_data.update(
             self._get_standardized_metadata())
         instance_data['ds']['_doc'] = EXPERIMENTAL_TEXT
+        # Add merged cloud.cfg and sys info for jinja templates and cli query
```

Comment:
Do we need an SRU: blocker tag here?

Reply:
I don't think w eneed SRU blocker here, because it is config data additions (which we've crossed a couple of previous SRUs in instance-data.json when we added new v1 keys)

Reply:
Yes, I agree, the critical values are what go in v1: {}; they are new keys and do not modify older v1 keys.